storey/flow.py

```diff
@@ -1015,50 +973,63 @@ def _get_attributes_as_blob(self, aggregation_element):
     # }
     async def _load_aggregates_by_key(self, table_path, key):
-        if not self._client_session:
-            self._lazy_init()
+        self._lazy_init()
 
-        request_body = json.dumps({'AttributesToGet': '*'})
+        container, table_path = _split_path(table_path)
 
-        response = await self._client_session.put(f'{self._webapi_url}/{table_path}/{key}',
-                                                  headers=self._get_item_headers, data=request_body, ssl=False)
-        body = await response.text()
-        if response.status == 404:
+        response = await self._v3io_client.kv.get(container, table_path, key, raise_for_status=v3io.aio.dataplane.RaiseForStatus.never)
+        if response.status_code == 404:
             return None
-        elif response.status == 200:
-            parsed_response = _v3io_parse_get_item_response(body)
+        elif response.status_code == 200:
             res = {}
-            for name, value in parsed_response.items():
+            for name, value in response.output.item.items():
                 if name.startswith(self._aggregation_attribute_prefix):
                     res[name[len(self._aggregation_attribute_prefix):]] = pickle.loads(value)
 
             return res
         else:
-            raise V3ioError(f'Failed to get item. Response status code was {response.status}: {body}')
+            raise V3ioError(f'Failed to get item. Response status code was {response.status_code}: {response.body}')
 
     async def _load_by_key(self, table_path, key):
-        if not self._client_session:
-            self._lazy_init()
+        self._lazy_init()
 
-        request_body = json.dumps({'AttributesToGet': '*'})
+        container, table_path = _split_path(table_path)
 
-        response = await self._client_session.put(f'{self._webapi_url}/{table_path}/{key}',
-                                                  headers=self._get_item_headers, data=request_body, ssl=False)
-        body = await response.text()
-        if response.status == 404:
+        response = await self._v3io_client.kv.get(container, table_path, key, raise_for_status=v3io.aio.dataplane.RaiseForStatus.never)
+        if response.status_code == 404:
             return None
-        elif response.status == 200:
-            parsed_response = _v3io_parse_get_item_response(body)
-            for name in parsed_response:
+        elif response.status_code == 200:
+            parsed_response = response.output.item
+            for name in parsed_response.items():
                 if name.startswith(self._aggregation_attribute_prefix):
                     del parsed_response[name]
             return parsed_response
         else:
-            raise V3ioError(f'Failed to get item. Response status code was {response.status}: {body}')
+            raise V3ioError(f'Failed to get item. Response status code was {response.status_code}: {response.body}')
+
+    async def _describe(self, container, stream_path):
```

Comment:
Why half of the functions accept `container` and `path` and some a combined `path`?

Reply:
Because once again we're forcecd to make this separation 😿

Reply:
Why is that? it's redundant to split the path in the "calling" flows if it's only used here anyways?
Or alternatively if you want you can split that path always outside the driver.. no?